### DISCUSSION TITLE
Disabling a Reusable Workflow

### DISCUSSION THREAD

**Discussion by dumitraand** [community/community#37631] [Actions]
Hello! I've tried [disabling](https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow) a [Reusable Workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows) but it seems to still be triggered from the caller workflow.

Is this behaviour intended? I did not find any reference to this limitation anywhere in the documents. 

If it is indeed intended, is there no way to be able to do this? 

Use case:
In case of an incident I want to disable the workflow X, so I create it as a reusable workflow that is called from the main workflow. I still want all the main workflow to run but not X.

**Comment by ossanna16** [comment]
Hi there @dumitraand and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
